packages/client/hmi-client/src/components/responsive-matrix/matrix.vue

```diff
@@ -0,0 +1,902 @@
+<template>
+	<div class="matrix-container" ref="matrix_container">
+		<div class="matrix" ref="matrix">
+			<!-- <component
+				v-for="(selectedCell, idx) in selectedCellList"
+				:key="idx"
+				:is="getSelectedDrilldownType(selectedCell)"
+				:getSelectedCellStyle="getSelectedCellStyle"
+				:update="update"
+				:move="move"
+				:selectedCell="selectedCell"
+				:dataRowList="dataRowList"
+				:dataColList="dataColList"
+				:labelRowList="labelRowList"
+				:labelColList="labelColList"
+				:parameters="dataParametersArray"
+				:parametersMin="dataParametersMin"
+				:parametersMax="dataParametersMax"
+				:colorFn="getSelectedGraphColorFn(selectedCell)"
+				@click="selectedCellClick(idx)"
+			/> -->
+		</div>
+	</div>
+</template>
+
+<script lang="ts">
+import { nextTick, PropType } from 'vue';
+
+import chroma from 'chroma-js';
+import { Viewport } from 'pixi-viewport';
+import {
+	Application,
+	DisplayObject,
+	Geometry,
+	Texture,
+	Shader,
+	Mesh,
+	FederatedPointerEvent
+} from 'pixi.js';
+import {
+	SelectedCell,
+	SelectedCellValue,
+	CellStatus,
+	CellType,
+	Uniforms
+} from '@/types/ResponsiveMatrix';
+import { uint32ArrayToRedIntTex } from './pixi-utils';
+
+// import ResponsiveCellBarContainer from './cell-bar-container.vue';
+// import ResponsiveCellLineContainer from './cell-line-container.vue';
+
+import matrixVS from './matrix.vs.glsl';
+import matrixFS from './matrix.fs.glsl';
```

Comment:
Add a comment explaining what those file are doing.

Reply:
similar to another comment above, vertex and fragment shaders are well understood components of gl, not sure we need a gl tutorial imbedded in the code

Reply:
Perfect, a simple comment saying `// Vertex and Fragment Shaders` will be perfect!